src/components/structures/RoomView.tsx

```diff
@@ -1991,6 +2010,29 @@ export class RoomView extends React.Component<IRoomProps, IRoomState> {
         );
     }
 
+    private onSubmitAskToJoin = (reason?: string): void => {
```

Comment:
Can we add a JSDoc comment, same comment for the following functions in this file